src/updatedownloader.cpp

```diff
@@ -44,10 +41,7 @@ namespace winsparkle
                                   helpers
  *--------------------------------------------------------------------------*/
 
-namespace
-{
-
-std::wstring GetUniqueTempDirectoryPrefix()
+std::wstring winsparkle::GetUniqueTempDirectoryPrefix()
```

Comment:
Is name space designation `winsparkle::` really needed here when already inside the namespace?